Update blog post to include change in paths for kernel version 4.14 (#32011)
kubernetes/website: content/en/blog/_posts/2019-03-29-kube-proxy-subtleties-debugging-an-intermittent-connection-resets.md

However, for the users that are affected by this bug, there is a way to mitigate the
problem by applying the following rule in your cluster.

> Note :  With worker node kernel version 4.14 and above, the path mentioned in the script to set the sysctl might be different. Recommendation would be to test running the command `echo 1 > /proc/sys/net/ipv4/netfilter/ip_conntrack_tcp_be_liberal` on your worker node by using an SSH connection. If the command returns `No such file or directory`, that means the netfilter directory on your worker most likely uses the following sysctl path`/proc/sys/net/netfilter/nf_conntrack_tcp_be_liberal`. Update the command in the daemonset below to use the correct path. 

Reviewer: ```suggestion
{{< note >}}
With worker nodes on kernel version 4.14 and above, the path mentioned in the script to set the sysctl might be different. 
The recommendation is to connect to the worker node through SSH and run the following the command 
`echo 1 > /proc/sys/net/ipv4/netfilter/ip_conntrack_tcp_be_liberal`. 
If the command returns `No such file or directory`, that means the netfilter directory likely uses the sysctl path`/proc/sys/net/netfilter/nf_conntrack_tcp_be_liberal`. 
You must update the command in the DaemonSet below to use the correct path. 
{{< /note >}}
```

Just a suggestion to use the Hugo shortcode and other edits

Reviewer 2: For _blogs_, I'm actually OK with not using this kind of shortcode. I'd save the callout rendering for docs proper.

> Note :  With worker node kernel version 4.14 and above, the path mentioned in the script to set the sysctl might be different. Recommendation would be to test running the command `echo 1 > /proc/sys/net/ipv4/netfilter/ip_conntrack_tcp_be_liberal` on your worker node by using an SSH connection. If the command returns `No such file or directory`, that means the netfilter directory on your worker most likely uses the following sysctl path`/proc/sys/net/netfilter/nf_conntrack_tcp_be_liberal`. Update the command in the daemonset below to use the correct path. 

Reviewer: ```suggestion
Note: With worker nodes on kernel version 4.14 and above, the path mentioned in the script to set the sysctl might be different. 
The recommendation is to connect to the worker node through SSH and run the following the command 
`echo 1 > /proc/sys/net/ipv4/netfilter/ip_conntrack_tcp_be_liberal`. 
If the command returns `No such file or directory`, that means the netfilter directory likely uses the sysctl path`/proc/sys/net/netfilter/nf_conntrack_tcp_be_liberal`. 
You must update the command in the DaemonSet below to use the correct path. 
```

Reviewer: How about changing the note to _Update_ so it informs readers that this blog has been updated

Author: @reylejano Yeah I agree that would look better instead of a Note. What do you think @sftim ?

Reviewer 2: I like that suggestion. Did anyone check in with the original author about whether this update is a good fit and how important it is to update the article?
